.github/workflows/release-apps.yml

```diff
@@ -0,0 +1,52 @@
+name: Release Apps
+
+on:
+  release:
+    types: [ published ]
+  push:
+    branches: [ main ]
+  pull_request_target:
+    types: [ opened, edited, synchronize, reopened ]
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Release Apps
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        app: [ legacy-api, ocean-api ]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@27d0a4f181a40b142cce983c5393082c365d1480
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@94ab11c41e45d028884a99163086648e898eed25
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@6af3c118c8376c675363897acf1757f7a9be6583
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Resolve Tags
+        uses: actions/github-script@9ac08808f993958e9de277fe43a64532a609130e
+        id: tags
+        env:
+          APP: ${{ matrix.app }}
+        with:
+          script: return require('./.github/scripts/release-tags.js')({ context })
+          result-encoding: string
+      
+      - name: Build & Publish
+        uses: docker/build-push-action@7f9d37fa544684fb73bfe4835ed7214c255ce02b
+        with:
+          push: true
+          build-args: APP=${{ matrix.app }}
+          platforms: |
+            linux/amd64
+            linux/arm64
+          tags: ${{ steps.tags.outputs.result }}
```

Comment:
For PR we should add something like this, easily allow the reader to grab the hash for testing.

via: `if: github.event_name == 'pull_request'`

https://github.com/DeFiCh/wallet/blob/df1696ccb922541c9b279944abb39d3bf4ee8b10/.github/workflows/expo-preview.yml#L35-L44

Reply:
yeah, that makes sense, will update.